### DISCUSSION TITLE
Should Nuxt be used with Vite or?

### DISCUSSION THREAD

**Discussion by AA1999** [nuxt/nuxt#23326] [Questions]
Hi
The [vite subpage for nuxt ](https://vite.nuxtjs.org/) states that Vite is experimental when used with Nuxt
However the last update is from 2 years ago, so I'm kinda questioning the reliability of this.

When you do `pnpx nuxi@latest init <project-name>` what does it use? Because it seems like it uses Vite but I don't see the vite config files either

**Comment by bikedawuwang** [comment]
I use Nuxt's default packaging tool Vite in the production environment. It bothers me. Every time I change a certain piece of code and build, all the hash values are changed. This makes it difficult to set up a cache for the user, so I Now using webpack5 as the build tool, but I think the compatibility between webpack5 and nuxt is still not good enough
